packages/create-workleap/spinner.ts

```diff
@@ -0,0 +1,40 @@
+import { spinner } from "@clack/prompts";
+
+const DEFAULT_SPINNER_MESSAGE = "Running...";
+let SPINNER_INSTANCE: {
+  start: (message: string) => void;
+  stop: (message: string) => void;
+};
```

Comment:
You shouldn't have to redefine the spinner's type. 

let SPINNER_INSTANCE: typeof spinner;

Reply:
Since the idea is basically wrapping the @clack/prompts, i would keep a similar synthax.

```
export const spinner = () => {
    return { 
        start(title?:string) {
            StartSpinner(title)
        },
        stop(title?:string) { 
            StopSpinner(title)
        }  
    } 
}
```

just don't export StartSpinner and StopSpinner

Reply:
My other comment is more important than this one i beleive